src/workflow/RunPreProcessMonthlyReports.hpp

```diff
@@ -3,243 +3,259 @@
+  SumOrAverage,                           !- Aggregation Type for Variable or Meter 1
+  ExteriorEquipment:NaturalGas,           !- Variable or Meter Name 2
+  SumOrAverage,                           !- Aggregation Type for Variable or Meter 2
+  Heating:NaturalGas,                     !- Variable or Meter Name 3
+  SumOrAverage,                           !- Aggregation Type for Variable or Meter 3
+  Cooling:NaturalGas,                     !- Variable or Meter Name 4
+  SumOrAverage,                           !- Aggregation Type for Variable or Meter 4
+  WaterSystems:NaturalGas,                !- Variable or Meter Name 5
+  SumOrAverage,                           !- Aggregation Type for Variable or Meter 5
+  Cogeneration:NaturalGas,                !- Variable or Meter Name 6
+  SumOrAverage;                           !- Aggregation Type for Variable or Meter 6
+)idf",
   R"idf(
-  Output:Table:Monthly,
-    Building Energy Performance - District Heating,  !- Name
-      2,                       !- Digits After Decimal
-      InteriorLights:DistrictHeating,  !- Variable or Meter 1 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 1
-      ExteriorLights:DistrictHeating,  !- Variable or Meter 2 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 2
-      InteriorEquipment:DistrictHeating,  !- Variable or Meter 3 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 3
-      ExteriorEquipment:DistrictHeating,  !- Variable or Meter 4 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 4
-      Fans:DistrictHeating,        !- Variable or Meter 5 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 5
-      Pumps:DistrictHeating,       !- Variable or Meter 6 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 6
-      Heating:DistrictHeating,     !- Variable or Meter 7 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 7
-      Cooling:DistrictHeating,     !- Variable or Meter 8 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 8
-      HeatRejection:DistrictHeating,  !- Variable or Meter 9 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 9
-      Humidifier:DistrictHeating,  !- Variable or Meter 10 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 10
-      HeatRecovery:DistrictHeating,!- Variable or Meter 11 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 11
-      WaterSystems:DistrictHeating,!- Variable or Meter 12 Name
-      SumOrAverage,            !- Aggregation Type for Variable or Meter 12
-      Cogeneration:DistrictHeating,!- Variable or Meter 13 Name
-      SumOrAverage;            !- Aggregation Type for Variable or Meter 13)idf",
+Output:Table:Monthly,
+  Building Energy Performance - District Heating Water, !- Name
```

Comment:
I renamed the table here. I think it's right, but would welcome comments from @DavidGoldwasser in particular

as indicated in the OP: AFAIK, this is `OpenStudio::EndUseFuelType::valueDescription` that is used by reporting measures so I made the change as well, perhaps some of them harcoded it to "District Heating", I'm not sure

Reply:
Added @joseph-robertson on to review of this PR as well